### PR TITLE
Hacktoberfest update the French translation [JENKINS-66681]

### PR DESCRIPTION
**Pull request by alauregaillard**
Same as https://github.com/jenkinsci/jenkins/pull/5776 but with a feature branch instead of fork:master (I hope...).

image

See [JENKINS-66681](https://issues.jenkins-ci.org/browse/JENKINS-66681).

### Proposed changelog entries

* Update the French translation in the legend.

### Proposed upgrade guidelines

N/A

### Submitter checklist

- [X] (If applicable) Jira issue is well described
- [x] Changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developer, depending on the change). [Examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)
  * Fill-in the `Proposed changelog entries` section only if there are breaking changes or other changes which may require extra steps from users during the upgrade
- [X] Appropriate autotests or explanation to why this change has no tests
- [x] For dependency updates: links to external changelogs and, if possible, full diffs

### Desired reviewers

@mention

### Maintainer checklist

Before the changes are marked as `ready-for-merge`: 

- [x] There are at least 2 approvals for the pull request and no outstanding requests for change
- [x] Conversations in the pull request are over OR it is explicit that a reviewer does not block the change
- [x] Changelog entries in the PR title and/or `Proposed changelog entries` are correct
- [x] Proper changelog labels are set so that the changelog can be generated automatically
- [x] If the change needs additional upgrade steps from users, `upgrade-guide-needed` label is set and there is a `Proposed upgrade guidelines` section in the PR title. ([example](https://github.com/jenkinsci/jenkins/pull/4387))
- [x] If it would make sense to backport the change to LTS, a Jira issue must exist, be a _Bug_ or _Improvement_, and be labeled as `lts-candidate` to be considered (see [query](https://issues.jenkins-ci.org/issues/?filter=12146)).

### CODE REVIEW

**Review by aheritier — APPROVED** [review]
LGTM

**Review by aHenryJard — APPROVED** [review]
Thank you for your PR :), All fine for me.

**Review by alecharp — APPROVED** [review]

File: core/src/main/resources/jenkins/model/Jenkins/legend_fr.properties
Comment (PierreBtz) [inline core/src/main/resources/jenkins/model/Jenkins/legend_fr.properties:26]: Is the double space on purpose in between the 2 sentences?

**Review by PierreBtz — APPROVED** [review]
lgtm once we sort out the double space question. Thanks for the contribution!

File: core/src/main/resources/jenkins/model/Jenkins/legend_fr.properties
Comment (alauregaillard) [reply core/src/main/resources/jenkins/model/Jenkins/legend_fr.properties]: No it's not. Thanks.

**Review by Wadeck — APPROVED** [review]
Bienvenue, merci pour la contribution :)

Merci d'avoir gardé la cohérence avec les entrées autour 👍

**Review by begault — APPROVED** [review]

**Review by oleg-nenashev — APPROVED** [review]
Thanks a lot for your pull request!

We may merge it in 24 hours if there is no negative feedback. Please see the [merge process documentation](https://github.com/jenkinsci/jenkins/blob/master/docs/MAINTAINERS.adoc#merge-process) for more information about the merge process

**Review by dduportal — APPROVED** [review]
Merci pour cette contribution! 🥳

**Review by P4uline — APPROVED** [review]
Merci !

**Pull request merged**
